funnel/assets/js/utils/markmap.js

```diff
@@ -3,6 +3,11 @@ const MarkmapEmbed = {
     $('.language-markmap').each(function embedMarkmap() {
       $(this).addClass('embed-added');
       $(this).find('code').addClass('markmap');
+      $(this).after(
+        $(
+          '<div>Click the empty white connectors <strong>&#9675;</strong> to close/collapse a node &amp; the filled coloured connectors <strong>&#9679;</strong> to open/expand a node.</div>'
```

Comment:
This needs `gettext` plus a term change from "click" to "tap" on mobile. IMO, best to not insert text into user content, and let the user figure this out.

Reply:
Got it